### DISCUSSION TITLE
The easiest way how to plot data?

### DISCUSSION THREAD

**Discussion by tesar-tech** [dotnet/interactive#1755] [Q&A] [answered]
I am struggling with this simple task. How to create plot and show it in dotnet interactive.
I tried this with XPlot.Plotly:

```csharp
#r "nuget: XPlot.Plotly.Interactive"
using XPlot.Plotly;
using XPlot.Plotly.Interactive;
var ch = Chart.Line(new[]{1,2,3,4});
ch
```
But it results in: 
![image](https://user-images.githubusercontent.com/33880579/142042805-32a197aa-8697-4011-9131-3cef2d3004ca.png)

```csharp
ch.Show();
```
opens html file with the plot, but I would like to keep as an output of cell.

XPlot.Plotly is not necessary to use. But based on my research it should be the easiest way..

What I am missing here?

**Comment by asmith-octopus** [comment, marked as answer]
I had this issue in the past and resolved it by adding `#i "nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json"` above `#r "nuget: XPlot.Plotly.Interactive"`.

Not sure if this should still be necessary. I found a reference to [this issue](https://github.com/fslaborg/XPlot/issues/162) which pointed me in the right direction.

**Reply by tesar-tech** [reply]
Well, it works. But the notebook needs to be saved first. Thanks.

**Comment by bmitc** [comment]
I might recommend using [Plotly.NET](https://plotly.net/) instead.
